pysap/data.py

```diff
@@ -119,6 +119,11 @@
                 "master/pysap-data/gen.mr"),
         "md5sum": None
     }
```

Comment:
I think this should fix the CI errors.

```suggestion
    },
```